src/main/java/jenkins/plugins/gerrit/client/GerritClient.java

```diff
@@ -0,0 +1,10 @@
+package jenkins.plugins.gerrit.client;
+
+import com.google.gerrit.extensions.api.changes.Changes;
+import jenkins.plugins.gerrit.GerritURI;
+
+public interface GerritClient {

```

Comment:
Hi,
Just a thought...
The restapi implementation is based on interfaces.
The GerritRestApi, GerritApi, Changes are interfaces.
Have you explored an option to implement these interfaces instead of having this new class?
I see you use the Changes as-is, so there should be no problem to use the GerritApi as well.
What do you think?

Reply:
Yes, that's actually my plan before I consider this PR done. I just wanted to post it as early as possible to get input!

I started without realizing this, but with Changes etc. I realized it and switched over. I just need to do it completely.